src/WAV.jl

```diff
@@ -761,18 +763,18 @@ function wavappend(samples::AbstractArray, io::IO)
         error("Number of channels do not match")
     end
 
-    # Compute data length of current chunk to-be-appended. 
+    # Compute data length of current chunk to-be-appended.
     data_length = size(samples, 1) * fmt.block_align
-    # Update `chunksize`: add length of new data. 
+    # Update `chunksize`: add length of new data.
     seek(io,4)
     write_le(io, convert(UInt32, chunk_size + data_length))
-    # Get `subchunk2size`. 
-    seek(io,40)
+    # Get `subchunk2size`.
+    seek(io, 24 + subchunk_size)
```

Comment:
This seems totally unrelated to the migration from Julia 0.6 to 0.7. How did this ever work with seeking to `40`? Usually the position should be `64`.

Reply:
This was submitted recently. I think that there is a unit test that goes with it.

Reply:
See pull request #61.

Reply:
Ah, yes, without this change, I was seeing the same test failures as in the CI log of that PR:
```
Test Failed
  Expression: length(y) == 2 * length(x)
   Evaluated: 8000 == 16000
ERROR: LoadError: There was an error during testing
while loading /home/travis/.julia/v0.6/WAV/test/runtests.jl, in expression starting on line 45
```
So apparently, #61 fixed a bug and added a test for it, but broke another test. With this update, they both pass. Without investigating closer, I guess the two tests append to WAV files with fmt chunks of different lengths, so a fixed value, 40 or 64, cannot be right for both.